tools/autotag/templates/rocm_changes/5.7.1.md

```diff
@@ -1,16 +1,15 @@
-<<<<<<< HEAD
-<!-- markdownlint-disable first-line-h1 -->
-<!-- markdownlint-disable no-duplicate-header -->
-
-### What's new in this release
-
-ROCm 5.7.1 is a point release with several bug fixes in the HIP runtime.
-
-#### HIP 5.7.1 (for ROCm 5.7.1)
-
-### Fixed defects
-
-=======
+<!-- markdownlint-disable first-line-h1 -->
+<!-- markdownlint-disable no-duplicate-header -->
+
+### What's new in this release
+
+ROCm 5.7.1 is a point release with several bug fixes in the HIP runtime.
+
+#### HIP 5.7.1 (for ROCm 5.7.1)
+
+### Fixed defects
+
+=======

```

Comment:
Thanks for catching this. Can we remove this as well? It's duplicated at the bottom.

Reply:
Thanks, I did not find this.

Reply:
Found more problems there. Now, should be good.